saas:build fails on `--no-dev`

saas:build fails right now when `composer install --no-dev` has been done. it cannot find bootstrap, as it's right now in require-dev 🤔 
https://github.com/symfony/demo/blob/main/composer.json#L66

Hello,

I'm facing the same issue with [d3052bc](https://github.com/symfony/demo/commit/d3052bc5a8574bbbc52340dca005a4c55f9cddef).

`composer install`:
```
$ composer install
No composer.lock file present. Updating dependencies to latest instead of installing from lock file. See https://getcomposer.org/install for more information.
Loading composer repositories with package information
Updating dependencies
Lock file operations: 119 installs, 0 updates, 0 removals
  - Locking composer/semver (3.4.3)
  - Locking dama/doctrine-test-bundle (v8.2.0)
  - Locking dflydev/dot-access-data (v3.0.3)
  - Locking doctrine/cache (2.2.0)
  - Locking doctrine/collections (2.2.2)
  - Locking doctrine/data-fixtures (1.7.0)
  - Locking doctrine/dbal (4.2.1)
  - Locking doctrine/deprecations (1.1.3)
  - Locking doctrine/doctrine-bundle (2.13.0)
  - Locking doctrine/doctrine-fixtures-bundle (3.6.1)
  - Locking doctrine/doctrine-migrations-bundle (3.3.1)
  - Locking doctrine/event-manager (2.0.1)
  - Locking doctrine/inflector (2.0.10)
  - Locking doctrine/instantiator (2.0.0)
  - Locking doctrine/lexer (3.0.1)
  - Locking doctrine/migrations (3.8.2)
  - Locking doctrine/orm (3.3.0)
  - Locking doctrine/persistence (3.3.3)
  - Locking doctrine/sql-formatter (1.5.1)
  - Locking egulias/email-validator (4.0.2)
  - Locking league/commonmark (2.5.3)
  - Locking league/config (v1.2.0)
  - Locking league/uri (7.4.1)
  - Locking league/uri-interfaces (7.4.1)
  - Locking masterminds/html5 (2.9.0)
  - Locking monolog/monolog (3.7.0)
  - Locking nette/schema (v1.3.2)
  - Locking nette/utils (v4.0.5)
  - Locking nikic/php-parser (v5.3.1)
  - Locking phpstan/extension-installer (1.4.3)
  - Locking phpstan/phpstan (1.12.7)
  - Locking phpstan/phpstan-doctrine (1.5.3)
  - Locking phpstan/phpstan-symfony (1.4.10)
  - Locking psr/cache (3.0.0)
  - Locking psr/clock (1.0.0)
  - Locking psr/container (2.0.2)
  - Locking psr/event-dispatcher (1.0.0)
  - Locking psr/http-factory (1.1.0)
  - Locking psr/http-message (2.0)
  - Locking psr/log (3.0.2)
  - Locking symfony/apache-pack (v1.0.1)
  - Locking symfony/asset (v7.1.6)
  - Locking symfony/asset-mapper (v7.1.6)
  - Locking symfony/browser-kit (v7.1.6)
  - Locking symfony/cache (v7.1.6)
  - Locking symfony/cache-contracts (v3.5.0)
  - Locking symfony/clock (v7.1.6)
  - Locking symfony/config (v7.1.6)
  - Locking symfony/console (v7.1.6)
  - Locking symfony/css-selector (v7.1.6)
  - Locking symfony/debug-bundle (v7.1.6)
  - Locking symfony/dependency-injection (v7.1.6)
  - Locking symfony/deprecation-contracts (v3.5.0)
  - Locking symfony/doctrine-bridge (v7.1.6)
  - Locking symfony/dom-crawler (v7.1.6)
  - Locking symfony/dotenv (v7.1.6)
  - Locking symfony/error-handler (v7.1.6)
  - Locking symfony/event-dispatcher (v7.1.6)
  - Locking symfony/event-dispatcher-contracts (v3.5.0)
  - Locking symfony/expression-language (v7.1.6)
  - Locking symfony/filesystem (v7.1.6)
  - Locking symfony/finder (v7.1.6)
  - Locking symfony/flex (v2.4.7)
  - Locking symfony/form (v7.1.6)
  - Locking symfony/framework-bundle (v7.1.6)
  - Locking symfony/html-sanitizer (v7.1.6)
  - Locking symfony/http-client (v7.1.6)
  - Locking symfony/http-client-contracts (v3.5.0)
  - Locking symfony/http-foundation (v7.1.6)
  - Locking symfony/http-kernel (v7.1.6)
  - Locking symfony/intl (v7.1.6)
  - Locking symfony/mailer (v7.1.6)
  - Locking symfony/maker-bundle (v1.61.0)
  - Locking symfony/mime (v7.1.6)
  - Locking symfony/monolog-bridge (v7.1.6)
  - Locking symfony/monolog-bundle (v3.10.0)
  - Locking symfony/options-resolver (v7.1.6)
  - Locking symfony/password-hasher (v7.1.6)
  - Locking symfony/phpunit-bridge (v7.1.6)
  - Locking symfony/polyfill-ctype (v1.31.0)
  - Locking symfony/polyfill-intl-grapheme (v1.31.0)
  - Locking symfony/polyfill-intl-icu (v1.31.0)
  - Locking symfony/polyfill-intl-idn (v1.31.0)
  - Locking symfony/polyfill-intl-messageformatter (v1.31.0)
  - Locking symfony/polyfill-intl-normalizer (v1.31.0)
  - Locking symfony/polyfill-mbstring (v1.31.0)
  - Locking symfony/polyfill-php83 (v1.31.0)
  - Locking symfony/process (v7.1.6)
  - Locking symfony/property-access (v7.1.6)
  - Locking symfony/property-info (v7.1.6)
  - Locking symfony/routing (v7.1.6)
  - Locking symfony/runtime (v7.1.6)
  - Locking symfony/security-bundle (v7.1.6)
  - Locking symfony/security-core (v7.1.6)
  - Locking symfony/security-csrf (v7.1.6)
  - Locking symfony/security-http (v7.1.6)
  - Locking symfony/service-contracts (v3.5.0)
  - Locking symfony/stimulus-bundle (v2.21.0)
  - Locking symfony/stopwatch (v7.1.6)
  - Locking symfony/string (v7.1.6)
  - Locking symfony/translation (v7.1.6)
  - Locking symfony/translation-contracts (v3.5.0)
  - Locking symfony/twig-bridge (v7.1.6)
  - Locking symfony/twig-bundle (v7.1.6)
  - Locking symfony/type-info (v7.1.6)
  - Locking symfony/ux-icons (v2.21.0)
  - Locking symfony/ux-live-component (v2.21.0)
  - Locking symfony/ux-twig-component (v2.21.0)
  - Locking symfony/validator (v7.1.6)
  - Locking symfony/var-dumper (v7.1.6)
  - Locking symfony/var-exporter (v7.1.6)
  - Locking symfony/web-profiler-bundle (v7.1.6)
  - Locking symfony/yaml (v7.1.6)
  - Locking symfonycasts/sass-bundle (v0.7.0)
  - Locking twbs/bootstrap (v5.3.3)
  - Locking twig/extra-bundle (v3.13.0)
  - Locking twig/intl-extra (v3.13.0)
  - Locking twig/markdown-extra (v3.13.0)
  - Locking twig/twig (v3.14.0)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 119 installs, 0 updates, 0 removals
  - Downloading symfony/dom-crawler (v7.1.6)
  - Downloading symfony/browser-kit (v7.1.6)
  - Downloading symfony/css-selector (v7.1.6)
  - Downloading symfony/debug-bundle (v7.1.6)
  - Downloading symfony/web-profiler-bundle (v7.1.6)
  - Installing symfony/flex (v2.4.7): Extracting archive
  - Installing symfony/runtime (v7.1.6): Extracting archive
  - Installing phpstan/phpstan (1.12.7): Extracting archive
  - Installing phpstan/extension-installer (1.4.3): Extracting archive
  - Installing composer/semver (3.4.3): Extracting archive
  - Installing symfony/deprecation-contracts (v3.5.0): Extracting archive
  - Installing symfony/routing (v7.1.6): Extracting archive
  - Installing symfony/polyfill-mbstring (v1.31.0): Extracting archive
  - Installing symfony/polyfill-ctype (v1.31.0): Extracting archive
  - Installing symfony/polyfill-php83 (v1.31.0): Extracting archive
  - Installing symfony/http-foundation (v7.1.6): Extracting archive
  - Installing psr/event-dispatcher (1.0.0): Extracting archive
  - Installing symfony/event-dispatcher-contracts (v3.5.0): Extracting archive
  - Installing symfony/event-dispatcher (v7.1.6): Extracting archive
  - Installing symfony/var-dumper (v7.1.6): Extracting archive
  - Installing psr/log (3.0.2): Extracting archive
  - Installing symfony/error-handler (v7.1.6): Extracting archive
  - Installing symfony/http-kernel (v7.1.6): Extracting archive
  - Installing symfony/finder (v7.1.6): Extracting archive
  - Installing symfony/filesystem (v7.1.6): Extracting archive
  - Installing symfony/var-exporter (v7.1.6): Extracting archive
  - Installing psr/container (2.0.2): Extracting archive
  - Installing symfony/service-contracts (v3.5.0): Extracting archive
  - Installing symfony/dependency-injection (v7.1.6): Extracting archive
  - Installing symfony/config (v7.1.6): Extracting archive
  - Installing psr/cache (3.0.0): Extracting archive
  - Installing symfony/cache-contracts (v3.5.0): Extracting archive
  - Installing symfony/cache (v7.1.6): Extracting archive
  - Installing symfony/framework-bundle (v7.1.6): Extracting archive
  - Installing doctrine/event-manager (2.0.1): Extracting archive
  - Installing doctrine/persistence (3.3.3): Extracting archive
  - Installing symfony/doctrine-bridge (v7.1.6): Extracting archive
  - Installing symfony/polyfill-intl-normalizer (v1.31.0): Extracting archive
  - Installing symfony/polyfill-intl-grapheme (v1.31.0): Extracting archive
  - Installing symfony/string (v7.1.6): Extracting archive
  - Installing symfony/console (v7.1.6): Extracting archive
  - Installing doctrine/sql-formatter (1.5.1): Extracting archive
  - Installing doctrine/deprecations (1.1.3): Extracting archive
  - Installing doctrine/dbal (4.2.1): Extracting archive
  - Installing doctrine/cache (2.2.0): Extracting archive
  - Installing doctrine/doctrine-bundle (2.13.0): Extracting archive
  - Installing dama/doctrine-test-bundle (v8.2.0): Extracting archive
  - Installing doctrine/lexer (3.0.1): Extracting archive
  - Installing doctrine/instantiator (2.0.0): Extracting archive
  - Installing doctrine/inflector (2.0.10): Extracting archive
  - Installing doctrine/collections (2.2.2): Extracting archive
  - Installing doctrine/orm (3.3.0): Extracting archive
  - Installing doctrine/data-fixtures (1.7.0): Extracting archive
  - Installing doctrine/doctrine-fixtures-bundle (3.6.1): Extracting archive
  - Installing symfony/stopwatch (v7.1.6): Extracting archive
  - Installing doctrine/migrations (3.8.2): Extracting archive
  - Installing doctrine/doctrine-migrations-bundle (3.3.1): Extracting archive
  - Installing nette/utils (v4.0.5): Extracting archive
  - Installing nette/schema (v1.3.2): Extracting archive
  - Installing dflydev/dot-access-data (v3.0.3): Extracting archive
  - Installing league/config (v1.2.0): Extracting archive
  - Installing league/commonmark (2.5.3): Extracting archive
  - Installing psr/http-message (2.0): Extracting archive
  - Installing psr/http-factory (1.1.0): Extracting archive
  - Installing league/uri-interfaces (7.4.1): Extracting archive
  - Installing phpstan/phpstan-doctrine (1.5.3): Extracting archive
  - Installing phpstan/phpstan-symfony (1.4.10): Extracting archive
  - Installing psr/clock (1.0.0): Extracting archive
  - Installing symfony/apache-pack (v1.0.1): Extracting archive
  - Installing symfony/asset (v7.1.6): Extracting archive
  - Installing masterminds/html5 (2.9.0): Extracting archive
  - Installing symfony/dom-crawler (v7.1.6): Extracting archive
  - Installing symfony/browser-kit (v7.1.6): Extracting archive
  - Installing symfony/css-selector (v7.1.6): Extracting archive
  - Installing twig/twig (v3.14.0): Extracting archive
  - Installing symfony/translation-contracts (v3.5.0): Extracting archive
  - Installing symfony/twig-bridge (v7.1.6): Extracting archive
  - Installing symfony/debug-bundle (v7.1.6): Extracting archive
  - Installing symfony/dotenv (v7.1.6): Extracting archive
  - Installing symfony/expression-language (v7.1.6): Extracting archive
  - Installing symfony/type-info (v7.1.6): Extracting archive
  - Installing symfony/property-info (v7.1.6): Extracting archive
  - Installing symfony/property-access (v7.1.6): Extracting archive
  - Installing symfony/polyfill-intl-icu (v1.31.0): Extracting archive
  - Installing symfony/options-resolver (v7.1.6): Extracting archive
  - Installing symfony/form (v7.1.6): Extracting archive
  - Installing league/uri (7.4.1): Extracting archive
  - Installing symfony/html-sanitizer (v7.1.6): Extracting archive
  - Installing symfony/http-client-contracts (v3.5.0): Extracting archive
  - Installing symfony/polyfill-intl-idn (v1.31.0): Extracting archive
  - Installing symfony/mime (v7.1.6): Extracting archive
  - Installing egulias/email-validator (4.0.2): Extracting archive
  - Installing symfony/mailer (v7.1.6): Extracting archive
  - Installing symfony/process (v7.1.6): Extracting archive
  - Installing nikic/php-parser (v5.3.1): Extracting archive
  - Installing symfony/maker-bundle (v1.61.0): Extracting archive
  - Installing monolog/monolog (3.7.0): Extracting archive
  - Installing symfony/monolog-bridge (v7.1.6): Extracting archive
  - Installing symfony/monolog-bundle (v3.10.0): Extracting archive
  - Installing symfony/phpunit-bridge (v7.1.6): Extracting archive
  - Installing symfony/polyfill-intl-messageformatter (v1.31.0): Extracting archive
  - Installing symfony/password-hasher (v7.1.6): Extracting archive
  - Installing symfony/security-core (v7.1.6): Extracting archive
  - Installing symfony/security-http (v7.1.6): Extracting archive
  - Installing symfony/security-csrf (v7.1.6): Extracting archive
  - Installing symfony/clock (v7.1.6): Extracting archive
  - Installing symfony/security-bundle (v7.1.6): Extracting archive
  - Installing symfony/translation (v7.1.6): Extracting archive
  - Installing symfony/twig-bundle (v7.1.6): Extracting archive
  - Installing symfony/ux-icons (v2.21.0): Extracting archive
  - Installing symfony/ux-twig-component (v2.21.0): Extracting archive
  - Installing symfony/stimulus-bundle (v2.21.0): Extracting archive
  - Installing symfony/ux-live-component (v2.21.0): Extracting archive
  - Installing symfony/validator (v7.1.6): Extracting archive
  - Installing symfony/web-profiler-bundle (v7.1.6): Extracting archive
  - Installing symfony/yaml (v7.1.6): Extracting archive
  - Installing symfony/http-client (v7.1.6): Extracting archive
  - Installing symfony/asset-mapper (v7.1.6): Extracting archive
  - Installing symfonycasts/sass-bundle (v0.7.0): Extracting archive
  - Installing twbs/bootstrap (v5.3.3): Extracting archive
  - Installing twig/extra-bundle (v3.13.0): Extracting archive
  - Installing symfony/intl (v7.1.6): Extracting archive
  - Installing twig/intl-extra (v3.13.0): Extracting archive
  - Installing twig/markdown-extra (v3.13.0): Extracting archive
Generating autoload files
97 packages you are using are looking for funding.
Use the `composer fund` command to find out more!

Run composer recipes at any time to see the status of your Symfony recipes.

Loading composer repositories with package information
Restricting packages listed in "symfony/symfony" to "7.2.*"
Updating dependencies
Lock file operations: 0 installs, 50 updates, 0 removals
  - Upgrading symfony/asset (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/asset-mapper (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/browser-kit (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/cache (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/clock (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/config (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/console (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/css-selector (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/debug-bundle (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/dependency-injection (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/doctrine-bridge (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/dom-crawler (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/dotenv (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/error-handler (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/event-dispatcher (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/expression-language (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/filesystem (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/finder (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/form (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/framework-bundle (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/html-sanitizer (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/http-client (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/http-foundation (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/http-kernel (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/intl (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/mailer (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/mime (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/monolog-bridge (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/options-resolver (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/password-hasher (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/process (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/property-access (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/property-info (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/routing (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/runtime (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/security-bundle (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/security-core (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/security-csrf (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/security-http (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/stopwatch (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/string (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/translation (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/twig-bridge (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/twig-bundle (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/type-info (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/validator (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/var-dumper (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/var-exporter (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/web-profiler-bundle (v7.1.6 => v7.2.0-BETA1)
  - Upgrading symfony/yaml (v7.1.6 => v7.2.0-BETA1)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 0 installs, 50 updates, 0 removals
  - Upgrading symfony/runtime (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/routing (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/http-foundation (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/event-dispatcher (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/var-dumper (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/error-handler (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/http-kernel (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/finder (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/filesystem (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/var-exporter (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/dependency-injection (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/config (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/cache (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/framework-bundle (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/doctrine-bridge (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/string (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/console (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/stopwatch (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/asset (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/dom-crawler (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/browser-kit (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/css-selector (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/twig-bridge (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/debug-bundle (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/dotenv (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/expression-language (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/type-info (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/property-info (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/property-access (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/options-resolver (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/form (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/html-sanitizer (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/mime (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/mailer (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/process (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/monolog-bridge (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/password-hasher (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/security-core (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/security-http (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/security-csrf (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/clock (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/security-bundle (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/translation (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/twig-bundle (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/validator (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/web-profiler-bundle (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/yaml (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/http-client (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/asset-mapper (v7.1.6 => v7.2.0-BETA1): Extracting archive
  - Upgrading symfony/intl (v7.1.6 => v7.2.0-BETA1): Extracting archive
Generating autoload files
97 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
phpstan/extension-installer: Extensions installed
> phpstan/phpstan-doctrine: installed
> phpstan/phpstan-symfony: installed

Symfony operations: 2 recipes (eedba42a4866a04d2ac3abd3d3cd4faf)
  - Configuring symfony/framework-bundle (>=7.2): From github.com/symfony/recipes:main
  - Configuring symfony/ux-icons (>=2.17): From github.com/symfony/recipes:main
Executing script cache:clear [OK]
Executing script assets:install public [OK]
Executing script importmap:install [OK]
Executing script sass:build [OK]
              
 What's next? 
              

Some files have been created and/or updated to configure your new packages.
Please review, edit and commit them: these files are yours.

 symfony/framework-bundle  instructions:

  * Run your application:
    1. Go to the project directory
    2. Create your code repository with the git init command
    3. Download the Symfony CLI at https://symfony.com/download to install a development web server

  * Read the documentation at https://symfony.com/doc


```

`composer install --no-dev-`:
```
$ composer install --no-dev
Installing dependencies from lock file
Verifying lock file contents can be installed on current platform.
Package operations: 0 installs, 0 updates, 16 removals
  - Removing twbs/bootstrap (v5.3.3)
  - Removing symfony/web-profiler-bundle (v7.2.0-BETA1)
  - Removing symfony/phpunit-bridge (v7.1.6)
  - Removing symfony/maker-bundle (v1.61.0)
  - Removing symfony/dom-crawler (v7.2.0-BETA1)
  - Removing symfony/debug-bundle (v7.2.0-BETA1)
  - Removing symfony/css-selector (v7.2.0-BETA1)
  - Removing symfony/browser-kit (v7.2.0-BETA1)
  - Removing phpstan/phpstan-symfony (1.4.10)
  - Removing phpstan/phpstan-doctrine (1.5.3)
  - Removing phpstan/phpstan (1.12.7)
  - Removing phpstan/extension-installer (1.4.3)
  - Removing nikic/php-parser (v5.3.1)
  - Removing doctrine/doctrine-fixtures-bundle (3.6.1)
  - Removing doctrine/data-fixtures (1.7.0)
  - Removing dama/doctrine-test-bundle (v8.2.0)
Generating autoload files
87 packages you are using are looking for funding.
Use the `composer fund` command to find out more!

Run composer recipes at any time to see the status of your Symfony recipes.

Executing script cache:clear [OK]
Executing script assets:install public [OK]
Executing script importmap:install [OK]
Executing script sass:build [KO]
 [KO]
Script sass:build returned with error code 1
!!  
!!   ! [NOTE] Executing Sass (pass -v to see more details).                         
!!  
!!  Error: Can't find stylesheet to import.
!!    ╷
!!  3 │ @import "../../vendor/twbs/bootstrap/scss/bootstrap.scss";
!!    │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
!!    ╵
!!    assets/styles/app.scss 3:9  root stylesheet
!!  
!!                                                                                  
!!   [ERROR] Sass build failed                                                      
!!                                                                                  
!!  
!!  
Script @auto-scripts was called via post-install-cmd
```

--- a/composer.json
+++ b/composer.json
@@ -47,6 +47,7 @@
         "symfony/validator": "^7",
         "symfony/yaml": "^7",
         "symfonycasts/sass-bundle": "^0.7",
+        "twbs/bootstrap": "^5",
         "twig/extra-bundle": "^3.3",
         "twig/intl-extra": "^3.3",
         "twig/markdown-extra": "^3.3"
@@ -64,8 +65,7 @@
         "symfony/maker-bundle": "^1.36",
         "symfony/phpunit-bridge": "^7",
         "symfony/stopwatch": "^7",
-        "symfony/web-profiler-bundle": "^7",
-        "twbs/bootstrap": "^5"
+        "symfony/web-profiler-bundle": "^7"
     },
     "config": {
         "allow-plugins": {

